etc/README.md

```diff
@@ -0,0 +1,46 @@
+# Installation instructions
+
+## bash
+
+Open your `.bashrc` file and add:
```

Comment:
This should be `~/.bashrc`. And, for OS X, we should suggest that they edit `~/.bash_profile` instead since that seems to be the convention right now.